packages/api/endpoints/s3credentials.js

```diff
@@ -40,9 +40,15 @@ async function requestTemporaryCredentialsFromNgap(username) {
 async function s3credentials(req, res) {
   const username = req.authorizedMetadata.userName;
   const credentials = await requestTemporaryCredentialsFromNgap(username);
-  return res.send(JSON.parse(credentials.Payload));
+  try {
+    return res.send(JSON.parse(credentials.Payload));
+  }
+  catch (error) {
+    log.error(`credentials.Payload: ${credentials.Payload}`);
```

Comment:
Is it safe to log this payload, since that means it will show up in Cloudwatch? Is this even intentional?

Reply:
Yeah it will show up in cloudwatch and only if the credentials we get back aren't parseable as a JSON object.  
I added this to try to discover what was going on when we got bad return values from NGAPs lambda.  I would expect only error messages here, but even if we logged the credentials, they are the timebased read-only credentials we're distributing.  Let me do a little test to see if we could actually ever get to this line.  I may have mistaken what the payload can look like and maybe I need to examine lambda's return value.

Reply:
Any update here?